health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/GeoPode.js

```
@@ -134,64 +182,65 @@ const GeoPode = () => {
+                        />
                     </div>
-                    <div style={{height:"2rem"}}></div>
+                    <div style={{ height: "2rem" }}></div>
                     <div>
                         {boundaryData.map((item, index) => (
                             <div>
-                                <div style={{fontWeight:"600", fontSize:"1.2rem"}}>
-                                    {item?.boundaryType}
+                                <div style={{ fontWeight: "600", fontSize: "1.2rem" }}>
+                                    {/* {item?.boundaryType} */}
+                                    {`${t((defaultHierarchyType + "_" + item?.boundaryType).toLowerCase().replace(/\s+/g, "_"))}`}
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)